src/lib/parts.ts

```diff
@@ -412,18 +424,25 @@ try {
       return false;
     }
   };
+  // tslint:disable-next-line:no-any
   window.addEventListener('test', options as any, options);
+  // tslint:disable-next-line:no-any
   window.removeEventListener('test', options as any, options);
 } catch (_e) {
 }
 
+
+type EventHandler = ((this: EventTarget, event: Event) => unknown)|{
```

Comment:
isn't there a type in lib.dom.ts for this already?

Reply:
Good point. Turns out it's EventListenerOrEventListenerObject